CONTRIBUTING.md

```diff
@@ -46,12 +51,17 @@ please ensure that:
    
    The documentation will be built/tested using Sphinx and should be free
    from errors and warnings.
+   Make sure you have `pandoc` installed:
```

Comment:
We could link here to the pandoc installing page for more info https://pandoc.org/installing.html

Reply:
Added the link